src/dapp/libexec/dapp/dapp-test-hevm

```diff
@@ -9,8 +9,11 @@ while [[ $# -gt 0 ]]; do
     -r|--regex)
       opts+=(--match "$1"); shift
       ;;
+    -vv)
+      opts+=(--verbose=2);
+      ;;
     -v|--verbose)
-      opts+=(--verbose);
+      opts+=(--verbose=1);
```

Comment:
This needs to be quoted, according to shellcheck.

Reply:
Or don't use the `=`